infra/Pos/Network/Yaml.hs

```diff
@@ -213,13 +221,16 @@ instance FromJSON AllStaticallyKnownPeers where
 instance FromJSON Topology where
   parseJSON = A.withObject "Topology" $ \obj -> do
       mNodes  <- obj .:? "nodes"
-      mRelays <- obj .:? "relays"
+      mWallet <- obj .:? "wallet"
       mP2p    <- obj .:? "p2p"
-      case (mNodes, mRelays, mP2p) of
+      case (mNodes, mWallet, mP2p) of
         (Just nodes, Nothing, Nothing) ->
             TopologyStatic <$> parseJSON nodes
-        (Nothing, Just relays, Nothing) ->
-            TopologyBehindNAT <$> parseJSON relays
+        (Nothing, Just wallet, Nothing) -> flip (A.withObject "wallet") wallet $ \walletObj -> do
+            relays    <- walletObj .:  "relays"
+            valency   <- walletObj .:? "valency"   .!= 1
```

Comment:
I believe this is the correct default, because this is what behind edge nodes do; we want to allow overriding this only for exchange nodes. But perhaps @dcoutts wants to change it for regular edge nodes too?